### PR TITLE
Update RT-, TWB-, Average Surround  theme & Deutsch locale

### PR DESCRIPTION
**Pull request by TooWaBoo**
Fixed some glitches in colorchooser
minor changes

TooWaGrey - Average Surround
Reduced contrast of highlighted buttons and changed blue Slider to grey.

Deutsch locale
Minor corrections

@Beep6581 
Please merge

### CODE REVIEW

**Comment by TooWaBoo** [comment]
Added workaround for Toolbar-Scroll-Bug in RT & TWB theme

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -181,7 +182,7 @@ FILEBROWSER_APPLYPROFILE;Profil anwenden
 FILEBROWSER_APPLYPROFILE_PARTIAL;Profil selektiv anwenden
 FILEBROWSER_AUTODARKFRAME;Automatisches Dunkelbild
 FILEBROWSER_AUTOFLATFIELD;Automatisches Weißbild
-FILEBROWSER_BROWSEPATHBUTTONHINT;Ausgewählter Pfad öffnen
+FILEBROWSER_BROWSEPATHBUTTONHINT;Ausgewählter Pfad öffnen.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Is this correct?

File: rtdata/languages/Deutsch [hunk 2]
```diff
@@ -1182,7 +1183,7 @@ PREFERENCES_GREYSC18;Yb = 18 CIE L#49
 PREFERENCES_GREYSCA;Automatisch
 PREFERENCES_HISTOGRAMPOSITIONLEFT;Histogramm linksseitig
 PREFERENCES_HISTOGRAMWORKING;Das Arbeitsprofil zur Darstellung des Haupthistogramms verwenden
-PREFERENCES_HISTOGRAM_TOOLTIP;Wenn aktiviert wird das Arbeitsprofil für die Darstellung\ndes Haupthistogramms verwendet, andernfalls das des\ngammakorrigierten Ausgangsprofil.
+PREFERENCES_HISTOGRAM_TOOLTIP;Wenn aktiviert wird das Arbeitsprofil für die Darstellung\ndes Haupthistogramms verwendet, andernfalls das des\ngammakorrigierten Ausgangsprofils.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Don't two verbs in a row demand a comma? Like "Wenn aktiviert, wird ..."

File: rtdata/languages/Deutsch (hunk 1 above)
Comment (heckflosse) [reply rtdata/languages/Deutsch]: `Ausgewählter Pfad öffnen.` ? really?

Isn't 
`Ausgewählten Pfad öffnen.` correct?

File: rtdata/languages/Deutsch (hunk 2 above)
Comment (heckflosse) [reply rtdata/languages/Deutsch]: :+1:

Comment (TooWaBoo) [reply rtdata/languages/Deutsch]: Thanks, fixed.

**Comment by Floessie** [comment]
:+1: for merge.

**Pull request merged**